src/components/molecules/plan/plan.scss

```
@@ -0,0 +1,52 @@
+@use "../../variables" as *;
+.plan {
+    display: flex;
+    position: relative;
+    text-align: center;
+    justify-content: center;
+    overflow-wrap: break-word;
+    font-size: $fontsize-s;
+    font-family: $fontfamily-gothic;
```

```suggestion
    text-align: center;
    overflow-wrap: break-word;
    font-size: $fontsize-s;
```
flexboxにしてしまうと、デフォルト値である`align-items: stretch;`によって中身が収縮してしまうので、ここではデフォルトの block要素のままで問題ないです。

参考：https://developer.mozilla.org/ja/docs/Web/CSS/align-items